.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="Rubocop" enabled="false" level="WARNING" enabled_by_default="false" />
+  </profile>
+</component>
```

Comment:
Files not related to the project should be added to `.gitignore`.
I would suggest deleting these files and adding `.idea/*` to `.gitignore`.

Reply:
Thanks @bolah2009!
We'll implement your suggestions in our next project